app/controllers/events_controller.rb

```diff
@@ -96,4 +120,58 @@ def date_from_form(date_info)
     def event_params
       params.require(:event).permit(:name, :description, :max_participants, :active, :kind, :organizer, :knowledge_level)
     end
+
+    # Generate all names to print from the query-params
+    #
+    # @return participant_names as array of strings
+    def badges_name_params
+      participant_names = []
```

Comment:
I still think this would be more elegant as a one-liner with `params.select { |key, value| key.include? "_print" }.map { |key, value| value }`. Don't you think? :slightly_smiling_face: